Extensions/SendTo/ClientPickerViewController.swift

```diff
@@ -36,6 +36,7 @@ class ClientPickerViewController: UITableViewController {
     var reloading = true
     var clients: [RemoteClient] = []
     var selectedClients = NSMutableSet()
+    var shareItem: ShareItem?
```

Comment:
Might be helpful to add a doc-style comment here so it's clear when/where this dangling property is used.